### ISSUE TITLE
Add creation loading panel to gui and code

### ISSUE THREAD

**Issue by jjonj** [Revolutionary-Games/Thrive#289]
The load button in the editor current does nothing.

**Comment by tjblazer85** [comment]
Hey jjonj, I'm on summer vacation and I wanted to help more on Thrive, Is this thread asking for a loading screen Art?

**Comment by Moopli** [comment]
Good question -- no it isn't. To clarify, this issue is asking for a modal to pop up when you click load in the editor (but only when accessed from main menu, loading a creature isn't available otherwise) which lists available microbes to load and play around with in editor. Eventually, this load button will take you to the Thrivepedia (or Thrive Content Library, or whatever) so you could load up whatever).

**Comment by tjblazer85** [comment]
Ok, thanks for the reply

**Comment by jjonj** [comment]
part of it is actually art, but I think oliveriver might be working on it

**Comment by Oliveriver** [comment]
Yeah, I'm a bit busy at the moment but it's top of my Thrive-related priorities when I get back.

**Comment by jjonj** [comment]
I now implemented saving/loading microbes using very ugly temporary gui. That should make sure we don't forget about this

**Comment by hhyyrylainen** [comment]
Perhaps related to this we could have a feature I've seen people on reddit being interested multiple times: saving creatures as png files? I think including a JSON dump in the metadata or after the image data as trailing bytes in the file. If we make this feature it *might* increase interest in Thrive.
